godot-macros/src/method_registration/register_method.rs

```diff
@@ -0,0 +1,171 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+use proc_macro2::{Ident, TokenStream as TokenStream2};
+use quote::quote;
+use venial::parse_declaration;
+
+use crate::method_registration::{
+    get_sig, get_signature_info, method_flags, wrap_with_unpacked_params, ReceiverType,
+};
+
+// Convenience function to wrap an object's method into a function pointer
+// that can be passed to the engine when registering a class.
+pub fn gdext_register_method(class_name: &Ident, method_signature: &TokenStream2) -> TokenStream2 {
+    let mut method_signature = parse_declaration(quote! { #method_signature {}})
+        .unwrap()
+        .as_function()
+        .unwrap()
+        .clone();
+
+    method_signature.body = None;
```

Comment:
Could you not reuse this method here?

https://github.com/godot-rust/gdext/blob/6ba5b01276639b31564c1d8a80b31c429c9787fb/godot-macros/src/util/mod.rs#L53-L61